docs/reST/ref/sprite.rst

```diff
@@ -637,10 +637,24 @@ Sprites are not thread safe. So lock them yourself if using threads.
       Default is 1000./80 where 80 is the fps I want to switch to full screen
       mode.  This method's name is a typo and should be fixed.
 
+      DEPRECATED: Use set_timing_threshold() instead.
```

Comment:
For this old function, only this line is enough, other docs can be removed. Additionally a `.. deprecated:: 2.1.1` line must be added here